src/mods/VR.cpp

```diff
@@ -2697,6 +2714,9 @@ void VR::openvr_input_to_re2_re3(REManagedObject* input_system) {
     // Change Ammo
     set_button_state(app::ropeway::InputDefine::Kind::CHANGE_BULLET, is_change_ammo_down);
 
+    // Toggle Flashlight
+    set_button_state(app::ropeway::InputDefine::Kind::FLASHLIGHT, is_toggle_flashlight_down);
```

Comment:
You cannot add non-existent bindings to the game's enums. Please remove these lines.

Reply:
ok no problem,  I didn't know what this line actually does.

Reply:
Yes, the app::ropeway::InputDefine::Kind enum belongs to the game. `set_button_state` is only for already existing in-game actions.